examples/game/alien_cake_addict.rs

```diff
@@ -191,49 +194,61 @@ fn move_player(
     keyboard_input: Res<Input<KeyCode>>,
     mut game: ResMut<Game>,
     mut transforms: Query<&mut Transform>,
+    time: Res<Time>,
 ) {
-    let mut moved = false;
-    let mut rotation = 0.0;
-    if keyboard_input.just_pressed(KeyCode::Up) {
-        if game.player.i < BOARD_SIZE_I - 1 {
-            game.player.i += 1;
+    game.player.move_cooldown = f32::max(game.player.move_cooldown - time.delta_seconds(), 0.);
```

Comment:
Why do this manually instead of using a Timer?

Reply:
Good point, I tested it and it seems I can get the same behavior just using a `Timer`.